core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataMap.java

```diff
@@ -223,17 +222,15 @@ protected short getBlockletId(DataMapRow dataMapRow) {
     return dataMapRow.getShort(BLOCKLET_ID_INDEX);
   }
 
-  protected ExtendedBlocklet createBlocklet(DataMapRow row, String fileName, short blockletId,
-      boolean useMinMaxForPruning) {
+  protected ExtendedBlocklet createBlocklet(DataMapRow row, String fileName, short blockletId) {
     if (isLegacyStore) {
-      return super.createBlocklet(row, fileName, blockletId, useMinMaxForPruning);
+      return super.createBlocklet(row, fileName, blockletId);
```

Comment:
useMinMaxForPruning flag is added to handle a different scenario where cache_level = blocklet and the filter column min max in not cached in driver. In that case the blocklet pruning for min/max needs to be done in each executor which is identified using this flag.

Reply:
yes, there are test errors about this problem